docs/templates/getting-started/faq.md

```diff
@@ -54,6 +57,21 @@ theano.config.floatX = 'float32'
 
 ---
 
+### What does \["sample", "batch", "epoch"\] mean?
+
+Below are some common definitions that are necessary to know and understand to correctly utilize Keras:
+
+- **Sample**: one element of a dataset.
+  - *Example:* one image is a **sample** in a convolutional network
+  - *Example:* one audio file is a **sample** for a speech recognition model
+- **Batch**: a set of *N* samples. The samples in a **batch** are run in parallel and, if training, result in only 1 update to the model.
+  - A **batch** generally approximates the distribution of the input data better than a single input. The larger the batch, the better the approximation; however, it is also true that the batch will take longer to processes and will still result in only 1 update.
+- **Epoch**: an arbitrary cutoff, generally defined by a count of samples, to separate training into distinct phases. Generally, an **epoch** is considered one pass over the available training data.
```

Comment:
Maybe add that the concept of "epoch" is important mainly for logging, callbacks, and evaluation. The call is to be able to log/call callbacks/evaluate from time to time, typically after one pass on the dataset.